src/components/movie/Movie.jsx

```diff
@@ -0,0 +1,23 @@
+import { Link } from "react-router-dom";
+
+const Movie = ({ movies }) => {
+  return (
+    <>
```

Comment:
is this necessary? Does it give an error without this?

Reply:
Yes, it gives an error with this because in return statement we can return only single element.